hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2346,6 +2368,18 @@
       stage: V2
       tests: ["chip_sw_csrng_kat_test"]
     }
+    {
+      name: chip_sw_csrng_edn_error
+      desc: '''Verify the outcome of an error response generated by CSRNG when processing an EDN
+               request.
+
+              - Inject a fault in CSRNG while it is processing a request from EDN so that it returns
+                an error response to EDN.
```

Comment:
Currently, CSRNG does not respond with an error status, see issue: https://github.com/lowRISC/opentitan/issues/16516